src/settings/AddTimerSchema.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import NumberInput from './NumberInput';
+import { useTimerSchemaReducer } from './useTimerSchemaReducer';
+
+export default function AddTimerSchema() {
+  const {
+    schemaName,
+    setSchemaName,
+    timerDuration,
+    setTimerDuration,
+    shortBreakDuration,
+    setShortBreakDuration,
+    longBreakDuration,
+    setLongBreakDuration,
+    elapsedTimersToLongBreak,
+    setElapsedTimersToLongBreak,
+    resetValuesToDefault,
+  } = useTimerSchemaReducer();
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    // Add to local Storage and reset
+    e.preventDefault();
+
+    resetValuesToDefault();
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <h3>Add Timer Option</h3>
+      <input
+        type='text'
+        id='Set timer schema name'
```

Comment:
id is not presented to screen readers, the value should make it easy to identify it in code.